null_safety_examples/layout/base/README.md

```diff
@@ -0,0 +1,16 @@
+# base
+
+A new Flutter project.
+
+## Getting Started
+
+This project is a starting point for a Flutter application.
+
+A few resources to get you started if this is your first Flutter project:
+
+- [Lab: Write your first Flutter app](https://flutter.dev/docs/get-started/codelab)
```

Comment:
Use relative links:  /docs/get-started/codelab and /docs/cookbook.

Reply:
Done.